services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: None = None) -> None:

```

Comment:
The parameter `session_date` should have a type hint of `Optional[date]` instead of `None`. The return type should be `QuerySet` instead of `None` since the function returns a queryset of movie sessions.